graphics/fontforge/Portfile

```diff
@@ -93,67 +46,38 @@ depends_lib \
                 port:libuninameslist \
                 port:libxml2 \
                 port:tiff \
-                port:xorg-libXi \
-                port:xorg-libXt \
-                port:xorg-libxkbui \
-                port:zlib
-
-use_autoconf    yes
+                port:zlib \
+                port:libspiro \
+                port:pango \
+                port:cairo
 
 configure.args  --mandir=${prefix}/share/man \
-                --enable-type3 \
-                --enable-devicetables \
-                --enable-gb12345 \
-                --without-python \
-                --without-freetype-bytecode \
-                --without-pango \
-                --without-cairo \
-                --x-includes=${prefix}/include \
-                --x-libraries=${prefix}/lib
+                --disable-python-scripting \
+                --disable-python-extension \
+                --without-x
```

Comment:
It would be awesome to make GUI work. If it doesn't, should we perhaps temporarily keep providing the old version with a working GUI?

Reply:
How can we keep 2 version of the same package?

Anyeay, the GUI part is really strange. Even with `--without-x`, the macOS GUI app is built.
I think we really have to figure out how the GUI is properly built.
Although I doubt that the devs might not be of much help. In on eof the tickets they mentioned that they basically can't develop on macOS anymore.